src/Microsoft.ML.SearchSpace/Converter/ParameterConverter.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
```

Comment:
We have a long standing issue with removing Newtonsoft.Json - [https://github.com/dotnet/machinelearning/issues/4359](https://github.com/dotnet/machinelearning/issues/4359). Can we just use System.Text.Json here from the beginning? That way we don't need to convert it in the future?

Reply:
`Parameter` class depends on `Json.Net` heavily so it'll be a big change. I'd prefer to save conversion work till when we start #4359

Reply:
You are introducing the `Parameter` class with this change. That's why I'm making the comment that we might as well not add "debt" with new functionality. Are there features in Json.Net that you are using that aren't available in System.Text.Json?

Reply:
`Parameter` and its converter class are using `System.Text.Json` now

I also upgrade Global variable `SystemTextJsonVersion` to 6.0.1, which is the latest version on nuget.org. It should be fine since no other projects are using that variable except this project.

Reply:
### Resolved